app/schema/api/v4/schema.rb

```diff
@@ -67,6 +67,10 @@ def user_activate_response
         required: %i[user] }
     end
 
+    def user_me_response
+      user_activate_response
```

Comment:
Can we copy-pasta the same schema info here? I think it's better than coupling these two API endpoints.